### PR TITLE
Convert the Units Registry conformance suite tests to pytest.

### PR DESCRIPTION
**Pull request by stevenbronson-wk**
#### Steps to Test
Run `pytest tests/integration_tests/validation/test_units_conformance_suite.py` and expect all 96 to pass.

**review**:
@Arelle/arelle

### CODE REVIEW

File: tests/integration_tests/validation/test_units_conformance_suite.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import glob
```
Comment (derekgengenbacher-wf) [inline tests/integration_tests/validation/test_units_conformance_suite.py:1]: Arbitrary line: I would change the name of this file to `test_utr_conformance_suite.py` to be more explicit.

File: tests/resources/conformance_suites/units/registry/utr-conf-cr-2013-05-17/2013-05-17/index.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (derekgengenbacher-wf) [inline tests/resources/conformance_suites/units/registry/utr-conf-cr-2013-05-17/2013-05-17/index.xml:1]: Aribtrary line: We dont want to commit the unzipped version of these conformance suites. Please update this so its just the zip.

**Comment by derekgengenbacher-wf** [comment]
This is also missing a stand alone script that we can run and generate log messages. See `scripts/conformance_suites/`.

File: arelle/CntlrCmdLine.py [hunk 3]
```diff
@@ -750,7 +750,7 @@ def monitorParentProcess():
             self.modelManager.disclosureSystem.select(None) # just load ordinary mappings
             self.modelManager.validateDisclosureSystem = False
         if options.utrUrl:  # override disclosureSystem utrUrl
-            self.modelManager.disclosureSystem.utrUrl = options.utrUrl
+            self.modelManager.disclosureSystem.utrUrl = [options.utrUrl]
```
Comment (derekgengenbacher-wf) [inline arelle/CntlrCmdLine.py:753]: Why this change?

Comment (stevenbronson-wk) [reply arelle/CntlrCmdLine.py:753]: DisclosureSystem expects utrUrl to be a list.  Looks like https://github.com/Arelle/Arelle/commit/e15b0a6c92f0ed9717575ce2bcb342dd03fa13e1 made it a list and missed this option.

Comment (derekgengenbacher-wf) [reply arelle/CntlrCmdLine.py:753]: OK that makes sense because we added a FERC specific utr check since that taxonomy was using units that were not fully accepted into the UTR.

File: tests/integration_tests/validation/test_utr_conformance_suite.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+def test_utr_structure_malformed_utrs_conformance_suite(malformed_utr_file):
+    args = ARGS + [
+        # any valid file that refers to units, hopefully small, since it's not the point of the test
+        '--file', os.path.join(STRUCTURE_CONFORMANCE_SUITE, 'tests', '01-simple', 'simpleValid.xml'),
+        '--utrUrl', os.path.join(STRUCTURE_CONFORMANCE_SUITE_ZIP, malformed_utr_file),
+    ]
+    controller = parseAndRun(args)
+    errors = controller.modelManager.modelXbrl.errors
+    # when it can't find the file, it returns 'arelleUtrLoader:error'
+    assert errors and errors != ['arelleUtrLoader:error']
```
Comment (derekgengenbacher-wf) [inline tests/integration_tests/validation/test_utr_conformance_suite.py:65]: Nice!!!

**Review by derekgengenbacher-wf — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

**Review by sagesmith-wf — APPROVED** [review]

**Review by hermfischer-wf — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10 Testing locally succeeded. Successfully ran the UTR tests locally.
